src/TestFramework/MSTest.Core/Assertions/StringAssert.cs

```diff
@@ -88,7 +111,35 @@ public static void Contains(string value, string substring)
         /// </exception>
         public static void Contains(string value, string substring, string message)
         {
-            Contains(value, substring, message, null);
+            Contains(value, substring, message, StringComparison.Ordinal, null);
+        }
+
+        /// <summary>
+        /// Tests whether the specified string contains the specified substring
+        /// and throws an exception if the substring does not occur within the
+        /// test string.
+        /// </summary>
+        /// <param name="value">
+        /// The string that is expected to contain <paramref name="substring"/>.
+        /// </param>
+        /// <param name="substring">
+        /// The string expected to occur within <paramref name="value"/>.
+        /// </param>
+        /// <param name="message">
+        /// The message to include in the exception when <paramref name="substring"/>
+        /// is not in <paramref name="value"/>. The message is shown in
+        /// test results.
+        /// </param>
+        /// <param name="comparisonType">
+        /// The comparison method to compare strings <paramref name="comparisonType"/>.
+        /// </param>
+        /// <exception cref="AssertFailedException">
+        /// Thrown if <paramref name="substring"/> is not found in
+        /// <paramref name="value"/>.
+        /// </exception>
+        public static void Contains(string value, string substring, string message, StringComparison comparisonType)
+        {
+            Contains(value, substring, message, comparisonType, null);
```

Comment:
are we sending `null` to ` params object[] parameters`? That isn't needed nor recommend? 

(same for StartsWith and EndsWith)

Reply:
added